security/storage/google/public_buckets/README.md

```diff
@@ -1,38 +1,41 @@
-## Google Open Buckets Policy Template
+# Google Open Buckets Policy Template
 
-### What it does
+## What it does
 
 This Policy Template will check your account for Google Cloud Storage buckets with public permission.
 
-### Input Parameters
+## Input Parameters
 
 This policy has the following input parameters required when launching the policy.
 
 - *Email addresses of the recipients you wish to notify* - Email to alert when it finds google buckets that meet the criteria
-- *Google Cloud Project* - The Google Cloud Project to run this policy against.
 
-### Policy Actions
+## Policy Actions
 
 The following policy actions are taken on any resources found to be out of compliance.
 
 - Send an email report
 
-### Required Permissions
+## Prerequisites
 
-This policy requires permissions to access RightScale resources (credentials).  Before applying this policy add the following roles to the user applying the policy.  The roles should be applied to all Accounts where the policy will run or the Organization. For more information on modifying roles visit the [Governance Docs](https://docs.rightscale.com/cm/ref/user_roles.html)
+This policy uses [credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) for connecting to the cloud -- in order to apply this policy you must have a credential registered in the system that is compatible with this policy. If there are no credentials listed when you apply the policy, please contact your cloud admin and ask them to register a credential that is compatible with this policy. The information below should be consulted when creating the credential.
 
-- Cloud Management - credential_viewer or admin
-- Cloud Management - Observer
+## Credential configuration
 
-### Google Required Permissions
+For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed:
+
+Provider tag value to match this policy: `gce`
+
+Required permissions in the provider:
 
 - storage.buckets.list
 - storage.buckets.getIamPolicy2
+- Scope for the credential is <https://www.googleapis.com/auth/monitoring.write https://www.googleapis.com/auth/cloud-platform https://www.googleapis.com/auth/monitoring.read https://www.googleapis.com/auth/monitoring https://www.googleapis.com/auth/compute https://www.googleapis.com/auth/compute.readonly https://www.googleapis.com/auth/devstorage.full_control>
```

Comment:
scope is no longer needed.

Reply:
I have addressed this comment please review the changes.